container/cgroupfs/utils.go

```diff
@@ -0,0 +1,63 @@
+// +build linux
+
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cgroupfs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+)
+
+func parseEntry(line string) (name string, value uint64, err error) {
+	fields := strings.Fields(line)
+	if len(fields) != 2 {
+		return name, value, fmt.Errorf("Invalid format: %s", line)
+	}
+
+	value, err = strconv.ParseUint(fields[1], 10, 64)
+	if err != nil {
+		return name, value, err
```

Comment:
What do you think about adding to error message some info which will be helpful to point which value cannot be parsed to uint, for example:
```
fmt.Errorf("Cannot parse value for field %s, err=%s", field[0], err.Error())
``` 
or maybe put some log- it might be here or in a place where `parseEntry` is called.